<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>483</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">#minimizeButton {
  image: url(:/icon/asset/icon/minimize-button1.png);
}

#minimizeButton, #maximizeButton {
  background-color: transparent;
  border-radius: 0px;
}

#minimizeButton:hover, #maximizeButton:hover {
  border-style: none;
  background-color: #80c7c7c7;
}

#minimizeButton:pressed, #maximizeButton:pressed {
  border-style: none;
  background-color: #80808080;
}

#closeButton {
  image: url(:/icon/asset/icon/close-button1.png);
  background-color: transparent;
  border-radius: 0px;
}

#closeButton:hover {
  image: url(:/icon/asset/icon/close-button2.png);
  border-style: none;
  background-color: #e81123;
}

#closeButton:pressed {
  image: url(:/icon/asset/icon/close-button2.png);
  border-style: none;
  background-color: #8c0a15;
}

#label {
  color: &quot;#fefefe&quot;;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>262</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="minimizeButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>26</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Minimize</string>
       </property>
       <property name="icon">
        <iconset resource="../StarRailToolDesktop.qrc">
         <normaloff>:/asset/icon/minimize-button1.png</normaloff>:/asset/icon/minimize-button1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>34</width>
         <height>26</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="maximizeButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>26</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Maximize</string>
       </property>
       <property name="iconSize">
        <size>
         <width>34</width>
         <height>26</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>26</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset resource="../StarRailToolDesktop.qrc">
         <normaloff>:/asset/icon/close-button1.png</normaloff>:/asset/icon/close-button1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>34</width>
         <height>26</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="2">
      <widget class="QStackedWidget" name="stackedWidget"/>
     </item>
     <item row="0" column="0">
      <widget class="QScrollArea" name="scrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QScrollBar:vertical
{
  width: 4px;
  border:0px solid;
  border-radius: 2px;
  margin: 0px,0px,0px,0px;
  background-color: transparent;
  background:#F5F5F5;
}
 
QScrollBar:vertical:hover
{
  width: 4px;
  border:0px solid;
  margin: 0px,0px,0px,0px;
  background:#F5F5F5;
}
 
QScrollBar::handle:vertical
{
  width:4px;
  background:#E1E1E1;
  border-radius: 2px;
  height: 40px;
  
}
 
QScrollBar::handle:vertical:hover
{
  background:#E1E1E1;
  border-radius: 2px;
}
 
QScrollBar::add-line:vertical
{
  height:11px;
  background-color: transparent;
  subcontrol-position:bottom;
  border:0px solid;
  border-radius: 2px;
 
}
QScrollBar::sub-line:vertical
{
  height:11px;
  background-color: transparent;
  subcontrol-position:top;
  border:0px solid;
  border-radius: 2px;
}
 
QScrollBar::add-page:vertical
{
  background-color: #F5F5F5;
  border:0px solid;
  border-radius: 2px;
}
 
QScrollBar::sub-page:vertical
{
  background-color: #F5F5F5;
  border:0px solid;
  border-radius: 2px;
}
 
QScrollBar::up-arrow:vertical{
  border:0px solid;
  border-radius: 3px;
}
 
QScrollBar::down-arrow:vertical {
  border:0px solid;
  border-radius: 3px;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>118</width>
          <height>451</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="toolButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>主页</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icon/asset/icon/maximize-button2.png</normaloff>:/icon/asset/icon/maximize-button2.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>角色图鉴</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icon/asset/icon/maximize-button2.png</normaloff>:/icon/asset/icon/maximize-button2.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_3">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_4">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_5">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../StarRailToolDesktop.qrc"/>
 </resources>
 <connections/>
</ui>
